metabase/metabase
# Fix timestamp saving in events

Epic: https://github.com/metabase/metabase/issues/20289

How to test:
- Add or edit an event
- Make sure the timezone offset is set correctly in requests to the backend
- Make sure `time_matters` set to `true` only when time is set
